Back End/InsiteTeamTask/Repositories/AttendanceService.cs

```diff
@@ -0,0 +1,105 @@
+﻿using InsiteTeamTask.MockData;
+using InsiteTeamTask.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace InsiteTeamTask.Repositories
+{
+    public class AttendanceService
+    {
+        private readonly MockDataService _service;
+
+        public AttendanceService()
+        {
+            _service = new MockDataService();
+        }
+
+        public List<Attendance> GetAttendanceListFor(int seasonNumber, int gameNumber)
```

Comment:
Using `IEnumerable<Attendance>` here as a return type would match up to your controllers, and would reduce the changes you might have to make in future if requirements change